libs/scss-foundation/tests/typography/fonts.tests.scss

```diff
@@ -0,0 +1,75 @@
+@use 'sass-true/sass/true';
+@use '../../src/modules/typography/fonts';
+
+@include true.describe('define-font()') {
+
+	@include true.it('should set the font family') {
```

Comment:
We don't need this many tests for mixin as simple as current one.
Remember our talks about first counting the number of tests needed? Number of branching + special cases.
We have 3 ifs + special case might be the bigger picture (when everything's given) = 4 tests and we have 6 atm...

Reply:
Validation tests are missing (that needs to also be adjusted - 2 out of 3), which will now increase number of tests :)